jsk_hand/softhand_ros/README.md

```diff
@@ -0,0 +1,178 @@
+# softhand_ros
+
+ROS package for SoftHand
+
+## Note
+
+This package depends on the branch below.
+
+- [pazeshun/dynamixel_motor@gripper-v6-devel](https://github.com/pazeshun/dynamixel_motor/tree/gripper-v6-devel)
+
+## Installation
+
+### Workspace build
+
+```bash
+source /opt/ros/$ROS_DISTRO/setup.bash
+mkdir ~/softhand_ws/src -p
+cd ~/softhand_ws/src
+wget https://raw.githubusercontent.com/knorth55/softhand_ros/master/fc.rosinstall -O .rosinstall
+wstool up
+rosdep install --ignore-src --from-paths . -y -r -i
+cd ~/softhand_ws
+catkin build
+```
+
+### Udev installation
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+roscd softhand_ros
+sudo cp udev/*.rules /etc/udev/rules.d
+sudo service udev reload
+sudo service udev restart
+```
+
+## How to use
+
+### Launch softhand
+
+#### For softhand v1
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+# for left softhand v1
+roslaunch softhand_ros softhand_left.launch
+# for right softhand v1
+roslaunch softhand_ros softhand_right.launch
+```
+
+#### For softhand v2
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+# for left softhand v2
+roslaunch softhand_ros softhand_v2_left.launch
+# for right softhand v2
+roslaunch softhand_ros softhand_v2_right.launch
+```
+
+### Control softhand by euslisp
+
+#### For softhand v1
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+roscd softhand_ros/euslisp
+roseus softhand-interface.l
+# euslisp interactive mode
+# (softhand-init)
+# (send *ri* :start-grasp)
+# (send *ri* :stop-grasp)
+```
+
+#### For softhand v2
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+roscd softhand_ros/euslisp
+roseus softhand-v2-interface.l
+# euslisp interactive mode
+# (softhand-v2-init)
+# (send *ri* :close-thumb)
+# (send *ri* :open-thumb)
+# (send *ri* :start-grasp)
+# (send *ri* :stop-grasp)
+```
+
+## Softhand hardware installation
+
+### Set baud rate
+
+#### For softhand v1
+
+```bash
+# set baud rate from 57600 to 1000000
+rosrun dynamixel_driver set_servo_config.py -b 57600 -r 1 MOTOR_ID
```

Comment:
Is MOTOR_ID variable? If so, how about describe like `<MOTOR_ID>`. And how can we set MOTOR_ID in the first time?

Reply:
for the first time, we need to search with `info_dump.py` command.
But if it is new, the default value is `1`.

Reply:
I added.